<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWindowH</class>
 <widget class="QDialog" name="addWindowH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add an Hero</string>
  </property>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBut">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hero's Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Strenght</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Agility</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Intelligence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>base HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>base Armor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>base MP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Magical Resistence</string>
   </property>
  </widget>
  <widget class="QComboBox" name="skill1Box">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="skill2Box">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill3</string>
   </property>
  </widget>
  <widget class="QComboBox" name="skill3Box">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill4</string>
   </property>
  </widget>
  <widget class="QComboBox" name="skill4Box">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="baseAttackBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Attack</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="strBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="aglBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="intBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="mpBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="hpBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="levBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="phyBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="magBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
